packages/mermaid/src/styles.ts

```diff
@@ -73,8 +73,10 @@ const getStyles = (
 `;
 };
 
-export const addStylesForDiagram = (type: string, diagramTheme: unknown): void => {
-  themes[type] = diagramTheme;
+export const addStylesForDiagram = (type: string, diagramTheme?: unknown): void => {
```

Comment:
> Can you change `unknown` to the proper type?

I guess `string` should be fine.

> And also add it to `themes` at the top of the file?

It would be hard with the current approach. I think we should add something like `BaseStyleOptions` that contains common options, and make other diagram extends it. I guess `fontFamily` is common as far as I know.

```diff
- const themes: Record<string, any> = {};
+ const themes: Record<string, (options?: BaseStyleOptions) => string> = {};
```

What do you think? And create `diagramTheme.ts` file would be great.

Reply:
`unknown` type isn't possible because the parameter type of other function is `any`.